documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -705,6 +703,116 @@ See the {link-prefix}:{link-mongodb-connector}#mongodb-connector-properties[comp
 
 This configuration can be sent via POST to a running Kafka Connect service, which will then record the configuration and start up the one connector task that will connect to the MongoDB replica set or sharded cluster, assign tasks for each replica set, perform a snapshot if necessary, read the oplog, and record events to Kafka topics.
 
+[[mongodb-adding-connector-configuration]]
+=== Adding connector configuration
+
+ifdef::community[]
+To run a {prodname} MongoDB connector, create a connector configuration file and add it to your Kafka Connect cluster. 
```

Comment:
It's not really a file, rather a call of the REST API, as you describe further below.

Reply:
For each connector, I updated this to:
... create a connector configuration and add the configuration to your .....
If that's not right, let me know. 
I committed and pushed these updates.